Prototypes/BeleensMerge.cs

```diff
@@ -0,0 +1,497 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Legion/CoreLegion.cs
+//cs_include Scripts/Legion/LegionExcercise/LegionExercise3.cs
+//cs_include Scripts/Legion/LegionExcercise/LegionExercise4.cs
+//cs_include Scripts/Nation/CoreNation.cs
+//cs_include Scripts/Nation/Various/TarosManslayer.cs
+//cs_include Scripts/Nation/Various/PurifiedClaymoreOfDestiny.cs
+//cs_include Scripts/Story/ArtixWedding.cs
+
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class BeleensMerge
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new();
+    public CoreStory Story = new();
+    public CoreAdvanced Adv = new();
+    public static CoreAdvanced sAdv = new();
+    public CoreLegion CoreLegion = new();
+    public LegionExercise3 LegionExercise3 = new();
+    public LegionExercise4 LegionExercise4 = new();
+    public CoreNation CoreNation = new();
+    public TarosManslayer TarosManslayer = new();
+    public ArtixWedding ArtixWedding = new();
```

Comment:
if the 2nd part of the public is not used within the script, you do not need the public only the //cs_include for that specific script. please check the rest of your publics above here